healthyregions/SDOHPlace
# Issue 115 schema to single object

This PR is for #115 and #121 . It will combine all JSONs in the `meta` folder into one JSON to use and add a 'Back to Result' button in the extended drawer.

### How to Test

1. Run the app and navigate to the search page
2. Perform searches and run filters, nothing should change in terms of functions
3. Open http://localhost:3000/api/metadata and you should see the combined JSON file being used
4. Open a drawer by clicking the 'More Info' button from one of the result
5. You should see a 'Back to Results' button in the drawer
6. Click the Back to Results' button. You should see the drawer closed
